app/models/kuroko2/job_definition.rb

```diff
@@ -62,7 +62,7 @@ module PreventMultiStatus
   }
 
 
-  validates :name, length: { maximum: 40 }, presence: true
+  validates :name, length: { maximum: 180 }, presence: true
```

Comment:
This change isn't related to "Extending API"

Reply:
Yeah not exactly... we are using the names of rake tasks as the name of job definitions, and sometimes they are bigger than 40 ... since the constraint at the db level is 180 I am not really sure why this validation is much lower...

I couldn't find any reason for it to be lower?

Reply:
The length can be made longer and I also think 40 is too short for English.
Could you make another pull-request to make this longer?

Reply:
No problem

Reply:
I opened #130 with this change